### DISCUSSION TITLE
Bootscore in the wild

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#317] [Show and tell]
bootscore offers a page of 'examples'. But they are 'only' intended to demo the basic features of bootscore. (https://bootscore.me/theme/ and https://bootscore.me/plugins/ ) Bootscore is a very promising tool to setup WP / bootstrap based sites, especially, because it is really free software (not only free of charge). Thus, I am going to ask this question:
Is there any list of real world examples = initiated bootscore based sites where we could see, what's really possible with bootscore&wordpress&bootstrap?
Thanks for your answers...

**Comment by crftwrk** [comment]
That's a good question. There are a lot Bootscore based themes out there, but not all of them are collected. themesinfo.com has listed some user-created live-sites:

- https://themesinfo.com/bootscore-wordpress-theme-design-bfz51
- https://themesinfo.com/bootscore-main-wordpress-ecommerce-template-cogur/latest
- Earlier v5 versions https://themesinfo.com/bootscore-5-best-woocommerce-theme-cjx4h/latest


Let's use this discussion to share new sites. If you have a project you want to show, post here. Screenshots are welcome.

**Reply by kreincke** [reply]
wonderful, that helps a lot. Many thanks for the quick reply!

**Comment by crftwrk** [comment]
Let's share some sites I found in the web. This sites are **not** done by me. They are either bigger, well done or have interesting content.

![illy](https://user-images.githubusercontent.com/51531217/227985597-96860094-3508-43f9-ab9f-c06d9847b06e.png)
https://illy.sg/

![another-subculture](https://user-images.githubusercontent.com/51531217/227984667-cea5460f-4731-4b43-bc8e-00186a1ca491.png)
https://anothersubculture.co.uk/

![herodonuts](https://user-images.githubusercontent.com/51531217/227984910-39b40bb5-7378-4309-91ea-50763f6249e0.png)
https://hereodonuts.com/

![canabiz](https://user-images.githubusercontent.com/51531217/227985460-a36a26c4-5735-4b8e-b9f3-49f04da1efe4.png)
https://www.canabiz.cz/

![marion](https://user-images.githubusercontent.com/51531217/227985874-f1e8d3e5-a7b3-4554-ab17-f0d41d199341.png)
https://www.marionzimet.com/

![faz-buch](https://user-images.githubusercontent.com/51531217/227986167-303117ac-ce5c-4e19-91c9-3db3cc2640af.png)
https://fazbuch.de The FAZ is one of the leading german newspapers

![finanzszene](https://user-images.githubusercontent.com/51531217/227986433-da6ea445-db4a-4c30-a3db-1ef6957d536c.png)
https://finanz-szene.de/

![yanis](https://user-images.githubusercontent.com/51531217/227986944-b9e3b186-1fa0-4f51-8d25-0daeec10af14.png)
https://www.yanisvaroufakis.eu/ The former greek financial minister uses Bootscore as well

![promethean](https://user-images.githubusercontent.com/51531217/227986575-1de8e479-56b7-4e5b-a717-9f38bd3b5555.png)
https://www.prometheanworld.com/

![webmo](https://user-images.githubusercontent.com/51531217/227986824-290b64b2-e0bc-43a3-b21e-dfb503f23a96.png)
https://webmo.co.jp/

![yoga](https://user-images.githubusercontent.com/51531217/227987158-9cfc24de-e107-482f-828a-3e398bbe49b8.png)
https://selfnessyoga.hu/

![stratum](https://user-images.githubusercontent.com/51531217/227987271-3a9d7d7c-09a4-45c8-b47f-3ca7c72016ce.png)
https://stratum-international.com/

![cpi](https://user-images.githubusercontent.com/51531217/227987416-9f7713b8-7466-4aef-9f04-2ef431e4f394.png)
https://www.cpi.org/

![state](https://user-images.githubusercontent.com/51531217/227987491-c99671ae-812f-423a-9d34-e44ac28fff60.png)
https://statefreedomcaucus.org/

**Comment by crftwrk** [comment]
My first site in 2025: https://song-ev.de

| Before  | After |
| ------------- | ------------- |
| ![before](https://github.com/user-attachments/assets/4787896f-8aaa-4457-9dc3-75b3e7e7b497)  | ![after](https://github.com/user-attachments/assets/0f5c52af-c4af-4263-8c3e-1314c4a4f8a4)  |

**Comment by cphillipsusu** [comment]
https://ncademi.org/
<img width="1673" height="1276" alt="image" src="https://github.com/user-attachments/assets/68d8dc1d-d0cf-4c00-8c50-77b5c48f56f9" />

**Comment by crftwrk** [comment]
Some of my sites, no one of them uses template overrides. All done by filters and actions, included custom coded mega-menues:

https://vdzev.de
<img width="1992" height="1494" alt="vdzev de" src="https://github.com/user-attachments/assets/24ad885d-9f6e-401e-8900-61c3f095f6c4" />



https://intelligent-heizen.info
<img width="1992" height="1494" alt="ih" src="https://github.com/user-attachments/assets/91e089a5-03b2-456a-915a-8f00e2d445c1" />


https://www.maxmutzke.de
<img width="1992" height="1494" alt="maxmutzke" src="https://github.com/user-attachments/assets/88e62ae5-f25e-467c-b26e-f22a8287cbec" />


https://totounddermannimmond.de
<img width="1992" height="1494" alt="totounddermannimmond" src="https://github.com/user-attachments/assets/0e781018-4867-4942-a16d-3f48fc785785" />

https://studio-hinz.de
<img width="1992" height="1494" alt="studio-hinz" src="https://github.com/user-attachments/assets/9f182a47-58f8-4db0-9791-c513ae896498" />


https://ckriebel-consulting.de
<img width="1992" height="1494" alt="ckriebel" src="https://github.com/user-attachments/assets/9b80c665-e5fc-418f-9b94-1d377f097f41" />

https://sdl-online.de
<img width="1992" height="1494" alt="sdl-online" src="https://github.com/user-attachments/assets/66788000-62c2-4085-9ed3-bafb95995bd1" />
